packages/truffle/test/scenarios/commands/build.js

```diff
@@ -1,6 +1,6 @@
 const assert = require("assert");
-const CommandRunner = require("../commandrunner");
-const MemoryLogger = require("../memorylogger");
+const CommandRunner = require("../commandRunner");
+const MemoryLogger = require("../MemoryLogger");
```

Comment:
I thought this renaming was a stylistic choice but notice one is camel case and the other pascal case. Which do you prefer?

Reply:
I did it because `MemoryLogger` is basically a class and `CommandRunner` is more a utility. I'd be ok with renaming it `CommandRunner.js` if you prefer though.

Reply:
Seems reasonable to me.